package.json

```diff
@@ -26,13 +29,32 @@
   "scripts": {},
   "maintainers": [
     {
-      "name": "Eli Perelman",
+      "name": "eliperelman",
       "email": "eli@eliperelman.com"
     },
     {
-      "name": "Rob Wood",
+      "name": "rwood-moz",
       "email": "rwood@mozilla.com"
     }
   ],
-  "directories": {}
+  "directories": {},
+  "gitHead": "6e9ddcd5b7db2ada4bba417fcca0b897849444a7",
```

Comment:
Shouldn't this be the latest commit?
https://github.com/eliperelman/node-mozdevice/commit/7eb661b20d6b14ccc71e939df02c82360eb0225a

Er wait, or does this get set later to match the commit when this change will be set? (I don't know how this works)